### ISSUE TITLE
Where are spatial_transformer_1_loss and flow_loss defined?

### ISSUE THREAD

**Issue by elfprincess3** [voxelmorph/voxelmorph#103]
Hello author,
The results print three loss "loss: - spatial_transformer_1_loss:  - flow_loss: ". I want to know how spatial_transformer_1_loss and flow_loss defined, but I cannot find where they are defined.
Sorry I am not familiar with keras based on tensorflow. Thanks!

**Comment by adalca** [comment]
The first loss is the sum of the other two. 
The other two are defined in the `loss` argument to the `model.compile()` function call. 
We have several implementations of losses. For example, in `train.py` they are defined as `[data_loss, losses.Grad('l2').loss]`, where `data_loss` might be `ncc` or `mse` or similar. Please look through `train.py` and trace this backwards, it will also help you understand the code.

**Comment by elfprincess3** [comment]
@adalca Thanks！
